### PR TITLE
[EN] Forums (formerly "Beatmap_Forum")

### PR DESCRIPTION
**Pull request by MegaApplePi**
change `Beatmap_Forum` to `Forums` and redo page

### CODE REVIEW

**Comment by Shadowtail117** [comment]
Fourms or Forums?

**Comment by MegaApplePi** [comment]
`Forums` -- I didn't catch the typo on the commit message and that description for this pull request ¯\\\_(ツ)\_/¯

File: wiki/Forums/en.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Fourms
```
Comment (Shadowtail117) [inline wiki/Forums/en.md]: Okay, in the case you should change line 1 to match.

Comment (MegaApplePi) [inline wiki/Forums/en.md]: ok, fixed this and another instance

File: wiki/Forums/en.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Forums
+
+![osu!forums](main.jpg "Image of the osu!forums")
+
+The osu!forums is one of the main features of the osu!website.
```
Comment (TPGPL) [inline wiki/Forums/en.md]: osu!website -> osu!web?

Comment (MegaApplePi) [inline wiki/Forums/en.md]: ¯\\\_(ツ)\_/¯ ... hmm, probably should add that to the glossary :/

File: wiki/Forums/en.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Forums
+
+![osu!forums](main.jpg "Image of the osu!forums")
+
+The osu!forums is one of the main features of the osu!web.
+Here, you can make new threads or posts within discussions inside of the osu!community.
+
+## Directory
```
Comment (kj415j45) [inline wiki/Forums/en.md]: Directories?

Comment (MegaApplePi) [inline wiki/Forums/en.md]: `Subfourms`?

File: wiki/Forums/en.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
+# Forums
+
+![osu!forums](main.jpg "Image of the osu!forums")
+
+The osu!forums is one of the main features of the osu!web.
```
Comment (TPGPL) [inline wiki/Forums/en.md]: The osu!forums Is or Are?

Comment (MegaApplePi) [inline wiki/Forums/en.md]: 🤔 I'm not sure... there is only one fourm but when talking about the fourms as a whole, then you would add the `s` to fourm?

**Pull request merged**